core-java-io/src/test/resources/listFilesUnitTestFolder/country.txt

```diff
@@ -0,0 +1 @@
+<xml></xml>
```

Comment:
This appears to be an xml file, not a txt file. Can we adjust one or the other please?

Reply:
Ok done